components/Modal/Action.vue

```diff
@@ -258,13 +258,18 @@ export default {
     smallestAmount: SMALLEST,
     network,
     feeDenom: network.stakingDenom,
+    gasMultiplier: 1,
   }),
   computed: {
     ...mapState(['session']),
     ...mapState('data', ['balances', 'balancesLoaded']),
     ...mapState('ledger', ['transport']),
     networkFees() {
-      return fees.getFees(this.transactionData.type)
+      if (this.step === feeStep) {
+        return this.setNetworkFees(this.transactionData.type)
```

Comment:
Btw, notice that `setNetworkFees` has side-effects (setting  `gasEstimateMultiplier`), I don't think it is a good practice on computed function